docker/localnet/container-env/seed-0.env

```diff
@@ -0,0 +1,11 @@
+# Environment variables used by the container
+
+# General
```

Comment:
Since this is a seed, you're going to need P2P_SEED_MODE, otherwise this isn't a seed